src/utils.ts

```diff
@@ -1,4 +1,7 @@
 import fs from 'fs';
+import path from 'path';
+
+export const TYPE_PREFIX = 'gapi.client.';
```

Comment:
Not sure if this is the best place for this.

Reply:
It's your call, I'm ok with both old and new places.
Perhaps, this could be passed as cli argument, but it might be an overkill, I can't se why anyone would like to change this.

Reply:
Yeah, I think I'll just leave it here for now. It's referenced in some of the `bin/` scripts as well, so even if it was a CLI argument, we'd still have to export the default value.